transmart-data/updatedb/release-17.1-HYVE-5/postgres/to_date_data_type.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+DO LANGUAGE 'plpgsql' $$
+DECLARE
+cd_query varchar;
+BEGIN
+ cd_query := (select (string_agg('select ' || c_facttablecolumn || ' as concept_cd from ' || c_tablename || ' where ' || c_columnname || ' ' || c_operator || ' ''' || c_dimcode || ''' escape ''^'' ', ' union ') || ';') as query from i2b2metadata.i2b2_secure where c_visualattributes like '__D');
+ set search_path TO "$user", i2b2demodata;
+ EXECUTE format('CREATE TEMP TABLE tmp_concept_codes ON COMMIT DROP AS %s', cd_query);
```

Comment:
@forus Fails if there is no such concept in the database (e.g., on transmart.thehyve.net)